### ISSUE TITLE
'Sort By' changes format upon exit

### ISSUE THREAD

**Issue by JamieWright99** [ppy/osu#17925]
### Type

Game behaviour

### Bug description

The displayed filters on the 'sort by' section will change when you close the game, depending on what sort filter is chosen when you exit. The sort filter chosen upon exiting will be the first sort filter in the list when you reopen the game, but this changes the format and can hide filters you want to see. 

In the video the 'Title' sort filter is hidden when I reopen the game

### Screenshots or videos

https://streamable.com/8r63vc

### Version

2022.418.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/8538637/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8538638/updater.log)
[database.log](https://github.com/ppy/osu/files/8538639/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8538640/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8538641/network.log)
[performance.log](https://github.com/ppy/osu/files/8538642/performance.log)

**Comment by peppy** [comment]
To confirm, you'd still expect `Difficulty` to be first after restarting, but `Title` should be second in the list?

**Comment by peppy** [comment]
Right, this is happening because you had a non-standard sort mode selected (`Title`) at the start of the video, which returned to its default location after you deselected it.

This is working as intended. I guess you'd prefer that the options were sorted by recent usage rather than alphabetically?

**Comment by JamieWright99** [comment]
I normally sort my maps by title so having that be the first shown start filter is how I ordered them, but it changes when I exit with another filter. I'd personally like it to not change the shown sort choices at all, but if this is how you intended it to be then I guess this isn't a bug issue.

**Comment by peppy** [comment]
Let's keep this open and track the UX change. I think it's a worthwhile consideration.

**Comment by respektive** [comment]
I think recent usage is good. There's some filters i basically never use. It makes sense for them to hide and only show the ones you actually use frequently.

**Comment by Joehuu** [comment]
The filters are now purely dropdowns in song select v2, so I don't think this applies anymore, so closing.
